appveyor.yml

```diff
@@ -1,12 +1,12 @@
 version: '{build}'
 skip_tags: true
-image: Visual Studio 2015
+image: Visual Studio 2017
 configuration: Release
 install:
   - ps: mkdir -Force ".\build\" | Out-Null  
-  - ps: Invoke-WebRequest "https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1" -OutFile ".\build\installcli.ps1"
+  - ps: Invoke-WebRequest "https://dot.net/v1/dotnet-install.ps1" -OutFile ".\build\installcli.ps1"
```

Comment:
These shouldn't be needed on the VS 2017 AppVeyor image; there's an appveyor.yml and Build.ps1 in https://github.com/serilog/serilog-sinks-file that can both be used here as-is, without modification.

Reply:
Sure I'll update it.